src/kOS/Suffixed/VesselTarget.cs

```diff
@@ -422,7 +422,7 @@ private void InitializeSuffixes()
                 () => System.Math.Min(Vessel.distanceLandedPackThreshold, Vessel.distancePackThreshold), 
                 value => { Vessel.distanceLandedPackThreshold = Vessel.distancePackThreshold = value; }));
             AddSuffix("ISDEAD", new NoArgsSuffix<bool>(() => (Vessel.state == Vessel.State.DEAD) ));
-            AddSuffix("STATUS", new Suffix<String>(() => shared.Vessel.situation.ToString()));
+            AddSuffix("STATUS", new Suffix<String>(() => Vessel.situation.ToString()));
```

Comment:
I thought this fix had already been pulled into develop?  I find it strange that this is showing up as a diff.